build-system/config.js

```diff
@@ -62,7 +62,7 @@ const commonIntegrationTestPaths =
     initTestsPath.concat(fixturesExamplesPaths, builtRuntimePaths);
 
 const testPaths = commonIntegrationTestPaths.concat([
-  'test/**/*.js',
+  'test/*/!(e2e)/**/*.js',
```

Comment:
Looking at https://github.com/ampproject/amphtml/tree/master/test, there are other folders that could cause errors like this in future.

I think it should suffice to include just `'test/integration/**/*.js'` here. Can you see if that works?

Reply:
Sure, it works with `test/integration/*.js`. Made the change below!

Reply:
Hmm. I just realized `testPaths` includes all unit and integration tests, which is something we should probably deprecate. Let's revert to your original solution, and deal with this separately.